pkg/k8s/run.go

```diff
@@ -18,6 +18,17 @@ import (
 
 // Run runs scan on kubernetes cluster
 func Run(cliCtx *cli.Context) error {
+	// Full-cluster scanning with '--format table' without explicit '--report all' is not allowed so that it won't mess up user's terminal.
+	if cliCtx.String("report") == "all" &&
+		!cliCtx.IsSet("report") &&
+		cliCtx.String("format") == "table" &&
+		!cliCtx.Args().Present() {
```

Comment:
What if putting this condition after `cmd.InitOption`?

```suggestion
	if opt.Report == "all" &&
		!cliCtx.IsSet("report") &&
		opt.Format == "table" &&
		!cliCtx.Args().Present() {
```